extensions/podman/src/extension.ts

```diff
@@ -451,10 +452,56 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
 
   // Check on initial setup
   checkDisguisedPodmanSocket(provider);
-  // update the status of the provider if the socket is changed, created or deleted
+
+  // Update the status of the provider if the socket is changed, created or deleted
   disguisedPodmanSocketWatcher = setupDisguisedPodmanSocketWatcher(provider, getSocketPath());
   extensionContext.subscriptions.push(disguisedPodmanSocketWatcher);
 
+  // Compatibility mode status bar item
+  // only available for macOS (for now).
+  if (isMac()) {
+    // Get the socketCompatibilityClass for the current OS.
+    const socketCompatibilityMode = getSocketCompatibility();
+
+    // Create a status bar item to show the status of compatibility mode as well as
+    // create a command so when you can disable / enable compatibility mode
+    const statusBarItem = extensionApi.window.createStatusBarItem();
+    statusBarItem.text = 'Docker Compatibility';
+    statusBarItem.command = 'podman.socketCompatibilityMode';
+    statusBarItem.tooltip = 'Enable or disable Docker socket compatibility mode for Podman';
+    statusBarItem.iconClass = 'fa fa-plug';
+    statusBarItem.show();
+    extensionContext.subscriptions.push(statusBarItem);
+
+    // Create a modal dialog to ask the user if they want to enable or disable compatibility mode
+    extensionApi.commands.registerCommand('podman.socketCompatibilityMode', async () => {
+      // Manually check to see if the socket is disguised (this will be called when pressing the status bar item)
+      const isDisguisedPodmanSocket = await isDisguisedPodman();
+
+      if (!isDisguisedPodmanSocket) {
+        const result = await extensionApi.window.showInformationMessage(
+          `Do you want to automatically enable socket compatibility mode for Podman?\n\n${socketCompatibilityMode.details}`,
+          'Enable',
+          'Close',
+        );
```

Comment:
It looks like I have this dialog

![image](https://user-images.githubusercontent.com/436777/225746494-5ec7d20f-a1cf-488e-9a6d-f062cb46cc59.png)

Reply:
while if I replace `Close` by `Cancel` I have:
![image](https://user-images.githubusercontent.com/436777/225746765-c0e9b428-0216-4a1a-bd50-1b7fbd84f4dc.png)


so I have a primary action/button enable which looks nicer to me . WDYT ?